src/test/run-make-fulldeps/instrument-coverage/expected_coverage_summary.json

```diff
@@ -0,0 +1,163 @@
+{
+  "data": [
+    {
+      "files": [
+        {
+          "expansions": [],
+          "filename": "/usr/local/google/home/richkadel/rust/src/test/run-make-fulldeps/instrument-coverage/main.rs",
```

Comment:
I think it probably is NOT OK. I address this. Thanks!

Reply:
As mentioned in another comment today in response to @tmandry 's review, I decided to use relative paths, directly out of the `SourceMap`, instead of absolute paths like Clang does it. That should have resolved this problem. (Note, the `expected` file is a different format and has a different name now: https://github.com/rust-lang/rust/blob/892643031cb3b8566fae36103868d8ad25987b1e/src/test/run-make-fulldeps/instrument-coverage/expected_coverage.txt)